boards/common/minichlink.board.cmake

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Dhiru Kholia
```

Comment:
s/rogramming/programming/ in the commit message

Reply:
Thanks - fixed now.